index.html

```diff
@@ -52,13 +52,24 @@
         style: 'text-align: right;',
       },
     };
+    
     const redirects = [
-      ["#/", "#/README"], 
+      ["#", "#/README"], 
       ["", "#/README"], 
       ["#/ModerationAPI/Against", "#/Archive/ModerationAPI/Against"],
       ["#/GettingStarted", "#/GettingStarted/QuickStart"],
     ];
-    redirects.forEach(r => location.hash == r[0] ? location.hash = r[1] : {});
+    
+    function routeChange () {
+      redirects.forEach(([source, target]) => {
+        const sources = [source, `${source}/`];
+        if (!sources.includes(location.hash)) return;
+        
+        location.hash = target;
+      });
+    }
+    
+    window.addEventListener("hashchange", () => routeChange);
```

Comment:
This needs to be:
```javascript
    window.addEventListener("hashchange", routeChange);
    window.addEventListener("load", routeChange);
```

For two reasons, 1) the way the lambda is set up it never actually calls the function. "() => routerChange() " would be correct but lambda is unnecessary here, and 2) this only triggers on hash change, not page load, so if a user goes to an old link from a different domain then the routing lookup won't trigger.

Reply:
Oh I must've forgot to add an initial call, thank you for catching that.